### DISCUSSION TITLE
3.0.4版本下，将视频播放界面拖动到扩展屏，进行时间戳生成没有任何反应

### DISCUSSION THREAD

**Discussion by mdsnalw** [aidenlx/media-extended#302] [Q&A] [answered]
1. 在Media Extended的插件设置：

![image](https://github.com/PKM-er/media-extended/assets/57386018/ca825468-836c-4868-9088-bee4bfda044c)

2. 尝试的操作：
2.1 在主显示器下进行时间戳视频笔记，生成正常
![image](https://github.com/PKM-er/media-extended/assets/57386018/7b438cde-3a24-4796-8bec-fad9b5985e96)
2.2 将视频拖动到扩展屏后，通过Link with Tab，绑定对应已经打开的笔记视频，按下快捷键，主显示器窗口下的笔记无时间戳生成. 
![image](https://github.com/PKM-er/media-extended/assets/57386018/e1387175-80a9-4a07-8cfd-55ebf6e8791f)

**Comment by aidenlx** [comment, marked as answer]
1. 升级到最新版本
2. https://mx.pkmer.net/faq#restore-link-with-pane
3. v3新版的命令名称变了，你可能需要检查快捷键是否还生效
